README.md

```diff
@@ -79,6 +86,29 @@ you will need to copy the settings described below into both of your
 configurations, as keeping them separate means that they cannot follow each
 other automatically.
 
+#### Without flakes
+
+If you haven't enabled flakes yet or don't want to use this feature,
+you can configure Home Manager to use Stylix by importing the
+`stylix.nix` file below (using Home Manager's way of importing files
+`imports = [./stylix.nix]`)
+
+```nix
+{pkgs, ...}@args:
+
+{
+  stylix = {
+    image = ./wallpaper.jpg;
+  };
+
+  imports = [((import ../../stylix) args).hm];
+}
+```
+
+Feel free to adapt the two `import` statements at the end to fetch
+Stylix and [base16.nix](https://github.com/SenchoPens/base16.nix)
+directly from GitHub if that's what you prefer.
```

Comment:
```suggestion

```

This paragraph is no longer needed as this is done automatically.

Reply:
I rewrote it. The line I left is still relevant I think. The user has to choose whether to clone Stylix or to import it from GitHub.